pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -103,7 +103,7 @@ func (d *DependenciesDistributor) Start(ctx context.Context) error {
 	d.InformerManager.ForResource(resourceBindingGVR, bindingHandler)
 	d.resourceBindingLister = d.InformerManager.Lister(resourceBindingGVR)
 	resourceWorkerOptions := util.Options{
-		Name:          "resource detector",
+		Name:          "dependencies distributor resource detector",
```

Comment:
The name has been changed to [dependencies resource detector](https://github.com/karmada-io/karmada/blame/82fc37ea04aca442f378a013e8846104595037ea/pkg/dependenciesdistributor/dependencies_distributor.go#L601) by #3811, and released in v1.7.0.